docs/BREAKINGCHANGES.md

```diff
@@ -6,151 +8,125 @@ TODO
-TODO
+TODO: Still open
 
 
 ### fix get-item -literalpath a*b if `a*b` doesn't actually exist to return error [5197](https://github.com/PowerShell/PowerShell/issues/5197)
-TODO
+Previously, `-literalpath` given a wildcard will treat it the same as -path and if the wildcard found no files, it would silently exit. Correct behavior should be that -literalpath is literal so if the file doesn't exist, it should error. Fix is to also see if wildcard is being suppressed.
 
 
 ### don't insert linebreaks to output (except for tables) [5193](https://github.com/PowerShell/PowerShell/issues/5193)
-TODO
+TODO: Not sure what there is besides the title
 
 
 ### Don't wrap stderr as ErrorRecord [5190](https://github.com/PowerShell/PowerShell/issues/5190)
-TODO
+Keep stderr as string and don't wrap as ErrorRecord.  This means that errors from running pwsh.exe within pwsh are now just stderr strings even though they visually look like ErrorRecords so tests had to be changed.
 
 
 ### Passing a ScriptBlock (in syntax only) to native commands produces surprising results [5187](https://github.com/PowerShell/PowerShell/issues/5187)
-TODO
-
-
-### Make -NoTypeInformation Default on Export-Csv and ConvertTo-Csv [5164](https://github.com/PowerShell/PowerShell/issues/5164)
-TODO
-
-
-### Remove DCOM support from *-Computer cmdlets [5152](https://github.com/PowerShell/PowerShell/issues/5152)
-TODO
+TODO: still open
 
 
 ### Import-Csv Should apply PSTypeNames uppon import when Type Information is present in the CSV [5134](https://github.com/PowerShell/PowerShell/issues/5134)
-TODO
+Adds PSTypeName preservation on `Import-Csv` and `ConvertFrom-Csv`.
 
 
 ### -NoTypeInformation should be default on Export-Csv [5131](https://github.com/PowerShell/PowerShell/issues/5131)
-TODO
+Sets `-NoTypreInformation` as the default for `Export-Csv` and `ConvertTo-Csv`, hides the `-NoTypeInformation` parameter switch, adds `-IncludeTypeInformation` switch to `Export-Csv` and `ConvertTo-Csv` to enable legacy behavior, and provides a terminating error when both `-NoTypeInformation` and `-IncludeTypeInformation` are supplied.
 
 
 ### allow pscustomobject to have property with empty string name [5129](https://github.com/PowerShell/PowerShell/issues/5129)
-TODO
+TODO: Not taking the change, dup of [2820](https://github.com/PowerShell/PowerShell/pull/2820)
 
 
 ### StdErr/ErrorRecord should not be modified on output [5127](https://github.com/PowerShell/PowerShell/issues/5127)
-TODO
+TODO: Closed, not merged
 
 
 ### -Verbose should not override $ErrorActionPreference [5113](https://github.com/PowerShell/PowerShell/issues/5113)
-TODO
+`$ErrorActionPreference` is longer overwritten by the `-Verbose` or `-Debug` parameters.
 
 
 ### Web Cmdlets should warn when  legacy -Credential is sent over unencrypted connections [5112](https://github.com/PowerShell/PowerShell/issues/5112)
```

Comment:
Extra space between `when` and `legacy`.  However, we should probably just remove the word `legacy`